java/CMakeLists.txt

```diff
@@ -547,3 +668,20 @@ if(NOT MINGW)
     COMPILE_PDB_NAME ${ROCKSDBJNI_STATIC_LIB}.pdb
   )
 endif()
+
+if(WITH_LZ4)
+#  find_package(lz4 REQUIRED)
+#  add_definitions(-DLZ4)
+  include_directories(${lz4_INCLUDE_DIRS})
+#  list(APPEND THIRDPARTY_LIBS lz4::lz4)
+endif()
+
+
+
+#list( TRANSFORM JAVA_TEST_CLASSES REPLACE  "src/test/java/([^ ]*).java" "\\1" OUTPUT_VARIABLE testfilebase )
```

Comment:
a test here will not be used by CI, but can be used for development .

Reply:
What's the reason that this is commented out? It looks useful to me.